web/scripts/index.js

```diff
@@ -76,7 +77,7 @@ post = ( url, data ) => {
             console.log(xhr.responseText);
         }};
 
-    xhr.send( data );
+    xhr.send( JSON.stringify(data) );
```

Comment:
we should also check what the JSON looks like when the backend receives it, we only checked if the POST was working